ibm_mq/tests/common.py

```diff
@@ -181,12 +181,17 @@
     ('ibm_mq.queue.open_output_count', GAUGE),
     ('ibm_mq.queue.trigger_type', GAUGE),
     ('ibm_mq.queue.depth_percent', GAUGE),
+]
+
+RESET_QUEUE_METRICS = [
     ('ibm_mq.queue.high_q_depth', GAUGE),
     ('ibm_mq.queue.msg_deq_count', COUNT),
     ('ibm_mq.queue.msg_enq_count', COUNT),
     ('ibm_mq.queue.time_since_reset', COUNT),
 ]
 
+QUEUE_METRICS = DEFAULT_QUEUE_METRICS + RESET_QUEUE_METRICS
```

Comment:
Is `QUEUE_METRICS` used anywhere?

Reply:
Yes it's used here: https://github.com/DataDog/integrations-core/blob/master/ibm_mq/tests/test_ibm_mq_int.py#L299